make_m33_tutorial_datasets.py

```diff
@@ -0,0 +1,37 @@
+# Create the smaller cutouts of the M33 CO and HI cubes to host and
+# be available for download online for the tutorials.
+
+
+import astropy.units as u
+from spectral_cube import SpectralCube
+from pathlib import Path
+
+outputdir = Path("/home/ekoch/ownCloud/code_development/radio_astro_tools/")
```

Comment:
We should avoid any absolute paths in these scripts - I know you know this, but just making a comment so we don't forget before merging :wink:

Reply:
Thanks for pointing this out! I missed this on review.

Right now these big cubes aren't hosted anywhere. Should I just leave this as a generic `path/to/data` for the time being? I mostly wanted this script included in this PR so I would remember how I made all of the tutorial cubes.

Reply:
Maybe just use some generic relative paths? Or even the current directory. You could then just add a note that says the script assumes the files are in a directory called <...>

Reply:
Done!